main/Fileserver.c

```diff
@@ -6,13 +6,13 @@
 #include <sys/stat.h>
 #include <dirent.h>
 
-#include "esp_err.h"
-#include "esp_log.h"
-#include "esp_vfs.h"
-#include "esp_spiffs.h"
-#include "esp_http_server.h"
+#include <esp_err.h>
+#include <esp_log.h>
+#include <esp_vfs.h>
+#include <esp_spiffs.h>
+#include <esp_http_server.h>
 
-#define FILE_PATH_MAX (ESP_VFS_PATH_MAX + CONFIG_SPIFFS_OBJ_NAME_LEN)
+#define FILE_PATH_MAX (ESP_VFS_PATH_MAX + CONFIG_FATFS_MAX_LFN)
```

Comment:
Is that supposed to be an E?
```suggestion
#define FILE_PATH_MAX (ESP_VFS_PATH_MAX + CONFIG_FATFS_MAX_LEN)
```

Reply:
So it is, however that is not my doing. The mistake is a part of the tool chain and comes from an autogenerated file. ¯\_(ツ)_/¯

Reply:
Very sad